app/models/miq_action.rb

```diff
@@ -848,6 +848,44 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
+  def action_container_image_prevent(action, rec, inputs)
```

Comment:
@moolitayer `action_annotate_container_image_deny_execution` (unless it's too long). Maybe something shorter (but more meaningful than `image_prevent`).